test/compflags/bradc/minimalModules/minModIntRealString.chpl

```diff
@@ -0,0 +1,15 @@
+// Showcase what works with --minimal-modules and how to add more.
+
+extern proc printf(f:c_string, x...);
+proc +(a, b) return __primitive("+", a, b);
+
+var i: int;
+var r: real;
+var str = "Hello";
+var cs  = c"world";
+
+i = 5;
+r = 6.6;
+i = i + 7;
+
+printf("%ld %g  %s, %s!\n", i, r, str: c_string, cs);
```

Comment:
The %ld fails on Mac using clang...  :(

     /var/folders/69/jbk0y4q95lq8xhhwpqgwf_rh000bxs/T//chpl-bradc-73169.deleteme/minModIntRealString.c:19:29: error: format specifies type 'long' but the argument has type 'int64_t' (aka 'long long') [-Werror,-Wformat]
    > printf("%ld %g  %s, %s!\n", i_chpl, r_chpl, ret_chpl, cs_chpl);
    >         ~~~                 ^~~~~~
    >         %lld
    > 1 error generated.